locales/ru.catkeys

```diff
@@ -0,0 +1,39 @@
+1	English	application/x-vnd.Clipdinger	1363565741
+Entries fade every %A% minutes.\nThe maximal tint is reached after\n%B% minutes (in %C% steps)	SettingsWindow		Записи затеняются каждые %A% минут.\nМаксимальное затенение будет достигнуто\nчерез %B% минут (в %C% шагов)
+Title:	EditWindow		Заголовок:
+Clear history	MainWindow		Очистить историю
+About Clipdinger	MainWindow		О Clipdinger
+Add to favorites	ClipList		Добавить в избранное
+Remove favorite	ClipList		Удалить из избранного
+entries in the clipboard history	SettingsWindow		записей в истории буфера обмена
+Edit title	EditWindow		Редактировать заголовок
+History	MainWindow		История
+OK	EditWindow		OK
+Settings…	MainWindow		Настройки
+Saved favourites	MainWindow		Избранное
+Move down	MainWindow		Переместить ниже
+OK	SettingsWindow		OK
+Paste to Sprunge.us	MainWindow		Опубликовать на Sprunge.us
+Paste to Sprunge.us	ClipList		Опубликовать на Sprunge.us
+Entries don't fade over time.	SettingsWindow		Записи не затеняются со временем.
+App	MainWindow		Приложение
+Minimize	MainWindow		Свернуть
+Quit	MainWindow		Выход
+Edit title	ClipList		Редактировать заголовок
+Pause fading	MainWindow		Приостановить затенение
+Cancel	SettingsWindow		Отмена
+Clipdinger	System name		Clipdinger
+Fade history entries over time	SettingsWindow		Затенять записи в истории со временем
+Max. tint level	SettingsWindow		Максимальный уровень затенения
+Steps	SettingsWindow		Количество шагов
+Auto-paste	SettingsWindow		Автоматическая вставка
+Help	MainWindow		Справка
+Thank you	Application		Спасибо
+Delay	SettingsWindow		Задержка
+Clipdinger v0.5.1\n\twritten by Humdinger\n\tCopyright 2015-2016\n\nClipdinger provides a history of clippings of the system clipboard and lets you create favorites.\n\nPlease let me know of any bugs you find or features you miss. Contact info is in the ReadMe, see menu item 'Help'.	Application		Clipdinger v0.5.1\n\tнаписан Humdinger\n\tCopyright 2015-2016\n\nClipdinger предоставляет историю записей из системного буфера обмена и возможность их добвления в избранное.\n\nПожалуйста, сообщайте мне обо всех найденных вами ошибках или новых функциях, которые вам нужны.  Данные для связи находятся в "Справке", доступной через меню.
+Clip	MainWindow		Запись
+Remove clip	ClipList		Удалить запись
+Clipdinger settings	SettingsWindow		Настройки Clipdinger
+Cancel	EditWindow		Отмена
+Move up	MainWindow		Переместить выше
+Sprunge.us service not available.	MainWindow		Сервис Sprunge.us не доступен.
```

Comment:
Thanks. Corrected with last commit, but since it's nothing really major, and the message probably isn't seen that often, it'll have to wait for the next release. Missed v0.5.3 by minutes... :)